src/material/select/testing/select-harness.ts

```diff
@@ -107,6 +109,13 @@ export class MatSelectHarness extends MatFormFieldControlHarness {
     }
   }
 
+  /** Selects the options that match the passed-in filter. */
+  async select(filter: OptionHarnessFilters = {}): Promise<void> {
+    await this.open();
+    const options = await this.getOptions(filter);
```

Comment:
should we check if its multiple selection and if not just click the first one?

Reply:
Good point. Changed.